### PR TITLE
12_findTheOldest fix ambiguity and remove redundant test description

### PR DESCRIPTION
**Pull request by donRehan**
fix ambiguity in test description , old was 'finds the person with the greatest age if someone is still living.'
remove redundant test that checks for 'finds the person with the greatest age if the OLDEST is still living' since both this and the above test for ability to check if there is no death year field.

## Because

I had faced difficulty understanding the tests so wanted to make it easier for others to understand the test when they are learning

## This PR

- Remove one of the three tests as I think it is redundant
- Modify the test description of the last test to make it more concise

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `01_helloWorld: Update test cases`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If this PR includes any changes that affect the solution of an exercise, I've also updated the solution in the `/solutions` folder

### CODE REVIEW

**Comment by fabulousgk** [comment]
These tests actually check two seperate cases.  In the first case it verifies that the function returns the oldest person when there is a person still living in the set.  In the second case it makes sure that the function returns the oldest person when there is a person still living in the set AND that person is still living.

Why would you need both checks?  Perhaps someone would write a function that simply skips living people completely and therefore misses a case.  Or does not compute the living person correctly...etc.

**Comment by donRehan** [comment]
According to what I understand from you. We check for these two cases. 

> Oldest including living people. Oldest is the living person. 

This means we want to check if the code in the two cases
> doesn't Skip living people.
> doesn't Skip dead people.

Then I would like to change the comment to be as stated above.
I will be adjusting it for better readability once I have your confirmation.

File: 12_findTheOldest/findTheOldest.spec.js [hunk 1]
```diff
@@ -21,26 +21,7 @@ describe('findTheOldest', () => {
...
-      {
-        name: "Jane",
-        yearOfBirth: 1912,
-        yearOfDeath: 1941,
-      },
-    ]
-    expect(findTheOldest(people).name).toBe('Ray');
-  });
-  test.skip('finds the person with the greatest age if the OLDEST is still living', () => {
+  test('finds the oldest person if yearOfDeath field is not present', () => {
```
[the remark this answers is not in the document]
Comment (donRehan) [reply 12_findTheOldest/findTheOldest.spec.js:24]: Yeah this does alleviate the ambiguity here. Thank you ! 🙏

[the remark this answers is not in the document]
Comment (thatblindgeye) [reply 12_findTheOldest/findTheOldest.spec.js:24]: Didn't even realize @fabulousgk said what I had said first so credit goes to them 😆 

Would you mind making an update to this PR, @donRehan?

**Comment by thatblindgeye** [comment]
@donRehan just checking if you're able to make the updates as discussed in the above comment?

**Comment by donRehan** [comment]
> @donRehan just checking if you're able to make the updates as discussed in the above comment?

I apologize this is my first PR didn't know how to change it, but I believe I fixed it as per our discussion now.

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**